cosmwasm/packages/accounting/src/contract.rs

```diff
@@ -0,0 +1,968 @@
+use std::marker::PhantomData;
+
+use anyhow::{bail, ensure, Context};
+use cosmwasm_std::{CustomQuery, Deps, DepsMut, Event, Order, StdError, StdResult};
+use cw_storage_plus::Bound;
+
+use crate::{
+    msg::Instantiate,
+    state::{
+        account::{self, Balance},
+        transfer, Account, Kind, Modification, Transfer, ACCOUNTS, MODIFICATIONS, TRANSFERS,
+    },
+};
+
+pub fn instantiate<C: CustomQuery>(deps: DepsMut<C>, init: Instantiate) -> anyhow::Result<Event> {
```

Comment:
Since `C` here is parametric, might as well go with the default instantiation `Empty`?
```suggestion
pub fn instantiate(deps: DepsMut, init: Instantiate) -> anyhow::Result<Event> {
```

Reply:
Is it even worth keeping this instantiation logic in light of #1932?